3.10/quick-start-guide.md

```diff
@@ -21,6 +22,65 @@ update, replace, and remove documents using ArangoDB's web interface or a driver
 
 Learn more about ArangoDB's [data model and concepts](data-model-and-concepts.html).
 
+## Available Tools in ArangoDB
+
+ArangoDB provides various tools to help you manage and work with your data.
+
+- [ArangoDB Server](programs-arangod.html) is the core component of ArangoDB that
+  stores data and handles requets. You can communicate with the server through
+  the web interface, command-line interface, drivers, and REST API.
```

Comment:
I think it's important to mention the name of the executable (arangod) and clarify the difference between an ArangoDB server as in an instance of the database system (which can be comprised of multiple arangod processes), the server binary, and a server node. For clusters, you shouldn't communicate with any server node but only Coordinators - not something that necessarily needs to be mentioned here, but perhaps there is a wording that leaves this room for interpretation.